src/sentry/integrations/mixins/issues.py

```diff
@@ -379,3 +379,9 @@ def sync_status_inbound(self, issue_key: str, data: Mapping[str, Any]) -> None:
                 "data": data,
             }
         )
+
+    def migrate_issues(self):
+        """
+        Migrate the corresponding plugin's issues to the integration and disable the plugins.
+        """
+        raise NotImplementedError
```

Comment:
Since it's not actually an error that some integrations don't have this method, maybe replace this with `pass`